### PR TITLE
feat(git-hooks): add lint-php, lint-js and pre-commit scripts

### PR DESCRIPTION
**Pull request by Nevoss**

### CODE REVIEW

File: packages/git-hooks/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@elementor/git-hooks",
+  "version": "0.2.0",
+  "description": "Elementor git hooks",
+  "author": "Elementor Team",
+  "homepage": "https://github.com/elementor/elementor-editor-packages",
+  "license": "MIT",
```
Comment (matipojo) [inline packages/git-hooks/package.json:7]: GPL3+

File: packages/git-hooks/rollup.config.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+module.exports = ({ path, plugins: { commonjs, shebang } }) => {
```
Comment (matipojo) [inline packages/git-hooks/rollup.config.js:1]: ESlint for all files.

File: packages/git-hooks/src/index.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+			[kebabCase(key)]: script,
+		};
+	},
+	{}
+);
+
+const args = process.argv.slice(2);
+const scriptName = args[0];
+
+if (!optionalScripts.hasOwnProperty(scriptName)) {
```
Comment (matipojo) [inline packages/git-hooks/src/index.js:20]: Check has scripts.

File: packages/git-hooks/src/scripts/lint-js.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+		silent: true,
+	});
+
+	if (composerExists.code !== 0) {
+		error('npm is not exists. please install npm and run `npm install`.');
+	}
+
+	const files = stagedFiles.join(' ');
+
+	const lint = exec(`npm run lint -- ${files}`);
```
Comment (matipojo) [inline packages/git-hooks/src/scripts/lint-js.js:26]: Catch the stdout/stderror and throw a relevant error.

File: packages/git-hooks/src/utils/logs.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
+import chalk from 'chalk';
```
Comment (matipojo) [inline packages/git-hooks/src/utils/logs.js:1]: @efelementor Use it like here.

**Review by matipojo — CHANGES_REQUESTED** [review]

File: package-lock.json [hunk 6]
```diff
@@ -3642,9 +3642,9 @@
       },
```
Comment (matipojo) [inline package-lock.json:3642]: Remove.

**Review by matipojo — CHANGES_REQUESTED** [review]

**Review by matipojo — APPROVED** [review]

File: packages/git-hooks/.gitignore [hunk 7]
```diff
@@ -0,0 +1 @@
+bin
```
Comment (KingYes) [inline packages/git-hooks/.gitignore:1]:
```suggestion
bin/
```

**Review by KingYes — CHANGES_REQUESTED** [review]

**Pull request closed without merging**